stack/__main__.py

```diff
@@ -650,6 +657,117 @@ def add_access_group_cache_as_secret_member(secret, resource_prefix: str):
             member=pulumi.Output.concat('group:', release_access_group.group_key.id),
         )
 
+    if enable_egress_project:
+        if not release_bucket:
+            raise ValueError(
+                'Requested egress project, but no bucket is available to egress from'
+            )
```

Comment:
We could skip that, as the same logic will apply once we share `main` buckets (with Requester Pays enabled).

Reply:
Are you okay if I leave this here until that's implemented, already caught me once :)